lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -34,10 +37,20 @@ def to_native(self, course_overview):
                 kwargs={'course_id': course_id},
                 request=request
             )
+            try:
+                get_discussion_course_or_404(CourseKey.from_string(course_id), request.user)
```

Comment:
@nasthagiri 
Do we include complete tabs info (i.e. `CourseTabsList`) from course to course_overview or just a new field as list of all enabled tabs would work too?
Technically, it would as in mobile_api views we would have discussion tab in that list to know if we return discussion api url
and for discussion api course validation we are already working with course and we can use its tabs.is_enabled method to determin whether to raise not found error or not.
Logically, I wanted to know whether we need other information for future access or not?